thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml

```diff
@@ -32,23 +32,29 @@ post:
           authorizationRequestId: 33333333-0000-0000-0000-000000000000
           transactionRequestId: '00000000-0000-0000-0000-000000000000'
           challenge: '<base64 encoded binary - the encoded challenge>'
-          quote:
```

Comment:
remind me why `quote`'s fields are being brought up to root level?

Reply:
The CCB didn't like us disclosing private details about the quote that aren't relevant to the PISP. We also don't need things such as condition, ilppacket etc.